src/components/Preview.jsx

```diff
@@ -36,7 +40,11 @@ export default function Preview({
   const title = get(mostRecentCompiledProject, 'title', '');
 
   return (
-    <div className="preview output__item">
+    <div
+      className="output__row preview"
+      ref={onRef}
+      style={isConsoleOpen ? prefixAll({flex: outputColumnFlex[0]}) : null}
```

Comment:
In the other draggable flex implementations, the child components / flex items are passed their specific flex, rather than the entire data structure, which seems preferable. Here the `<Preview/>` is required to know that it’s the first flex item, which increases coupling.

Reply:
Great I will update this